fuzz/run.sh

```
@@ -0,0 +1,71 @@
+
+FUZZ_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+SECS="${1:?usage: run.sh <seconds> [jobs]}"
+JOBS="${2:-1}"
+
+FUZZ_BIN="${FUZZ_BIN:-$FUZZ_DIR/fuzz_str_to_char}"
+CORPUS_DIR="${CORPUS_DIR:-$FUZZ_DIR/corpus}"
+DICT="${DICT:-$FUZZ_DIR/fuzz.dict}"
+MAX_LEN="${MAX_LEN:-16384}"
+RSS_MB="${RSS_MB:-4096}"
+
+cd "$FUZZ_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)